.github/workflows/reusable-package-and-publish.yaml

```diff
@@ -59,10 +56,10 @@ jobs:
           python -m pip install -U build pip
           python -m build
 
-      # - name: Publish package to PyPI
-      #   # upload to PyPI on every release for a tag starting with 'v'
-      #   # if: github.event.action == 'published' && startsWith(github.event.release.tag_name, 'v')
-      #   uses: pypa/gh-action-pypi-publish@v1.8.10
-      #   with:
-      #     user: ${{ secrets.pypi_user }}
-      #       password: ${{ secrets.pypi_password }}
+        - name: Publish package to PyPI
+          # upload to PyPI on every release for a tag starting with 'v'
+          # if: github.event.action == 'published' && startsWith(github.event.release.tag_name, 'v')
+          uses: pypa/gh-action-pypi-publish@v1.8.10
+          with:
+            user: ${{ secrets.pypi_user }}
+            password: ${{ secrets.pypi_password }}
```

Comment:
Is this going to run on every repository? Should I start setting up PyPi projects for each repo?

Reply:
Well, it will just fail if it doesn't exist?  Do we want this to run on every repository?

Reply:
Probably eventually. I think we probably need a mechanism to turn it off for some repositories for now. Well, unless, is it okay for this to fail? It wouldn't result in a red "X" on a PR, it would just be a failure in the release process which is probably fine?

Reply:
Yeah, would just fail after version release PR is merged.  Would not block or break anything.

Reply:
Actually, it will currently never run, because we do not name our releases/tags starting with v.

Reply:
Let's just leave it for now. We should get this working soon. I need to get back to figuring out how to get akima86 to build as a wheel.